### ISSUE TITLE
optimizers.RMSprop

### ISSUE THREAD

**Issue by Raghav-Bell** [fchollet/deep-learning-with-python-notebooks#168]
I am trying some codes from your fantastic book, but got unconventional error . Please give an eye on it .
Thank you
model.compile(optimizer=optimizers.RMSprop(learning_rate=0.001), loss='binary_crossentropy' , metrics=['accuracy']) (on p-73)
**Error** : module 'keras.optimizers' has no attribute 'RMSprop'

**Comment by JonMetcalfe** [comment]
While I'm unsure about the entire story behind it, I think this comes from tensorflow acquiring keras?

Try replacing all imports that look like:
`from keras import _________`
to
`from tensorflow.keras import _________`

This resolved the issue for me!

**Comment by Raghav-Bell** [comment]
Yes, I got to know about this, 2 days before  . Thanks
